core/capabilities.js

```diff
@@ -44,6 +44,8 @@ define('core/capabilities', ['core/settings'], function(settings) {
         'debug': document.location.href.indexOf('dbg') >= 0,
         'debug_in_page': document.location.href.indexOf('dbginpage') >= 0,
         'console': window.console && typeof window.console.log === 'function',
+        'isPackagedApp': window.location.protocol === 'app:',
```

Comment:
I almost did that, but I didn't want it to be mixed up with the capability to install packaged apps (`packagedWebApps`). Any alternate suggestion ?